[2025-01-17T18:35:17.260+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-17T18:35:17.278+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_ELT_PostgreSQL.load_data manual__2025-01-17T18:35:07.326766+00:00 [queued]>
[2025-01-17T18:35:17.286+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_ELT_PostgreSQL.load_data manual__2025-01-17T18:35:07.326766+00:00 [queued]>
[2025-01-17T18:35:17.286+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-17T18:35:17.298+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data> on 2025-01-17 18:35:07.326766+00:00
[2025-01-17T18:35:17.307+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3587) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-17T18:35:17.308+0000] {standard_task_runner.py:72} INFO - Started process 3589 to run task
[2025-01-17T18:35:17.308+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DAG_ELT_PostgreSQL', 'load_data', 'manual__2025-01-17T18:35:07.326766+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/ETL_PostgreSQL.py', '--cfg-path', '/tmp/tmpxqbbikfr']
[2025-01-17T18:35:17.309+0000] {standard_task_runner.py:105} INFO - Job 52: Subtask load_data
[2025-01-17T18:35:17.349+0000] {task_command.py:467} INFO - Running <TaskInstance: DAG_ELT_PostgreSQL.load_data manual__2025-01-17T18:35:07.326766+00:00 [running]> on host 59ff16f1d4fb
[2025-01-17T18:35:17.624+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Wilfrido Rondon' AIRFLOW_CTX_DAG_ID='DAG_ELT_PostgreSQL' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-17T18:35:07.326766+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-17T18:35:07.326766+00:00'
[2025-01-17T18:35:17.625+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-17T18:35:17.625+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-17T18:35:17.625+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2025-01-17 18:35:17.278598+00:00
[2025-01-17T18:35:17.625+0000] {logging_mixin.py:190} INFO - Dag name:DAG_ELT_PostgreSQL and current dag run status:running
[2025-01-17T18:35:17.625+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-17T18:35:17.626+0000] {postgres.py:204} INFO - Running copy expert: COPY sales_db FROM STDIN, filename: /tmp/sales_db.csv
[2025-01-17T18:35:17.634+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn_local'
[2025-01-17T18:35:17.672+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-17T18:35:17.681+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-17T18:35:17.681+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=DAG_ELT_PostgreSQL, task_id=load_data, run_id=manual__2025-01-17T18:35:07.326766+00:00, execution_date=20250117T183507, start_date=20250117T183517, end_date=20250117T183517
[2025-01-17T18:35:17.689+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-17T18:35:17.689+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-17T18:35:17.690+0000] {logging_mixin.py:190} INFO - Dag name:DAG_ELT_PostgreSQL queued_at:2025-01-17 18:35:07.339472+00:00
[2025-01-17T18:35:17.690+0000] {logging_mixin.py:190} INFO - Task hostname:59ff16f1d4fb operator:PythonOperator
[2025-01-17T18:35:17.726+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-17T18:35:17.744+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-17T18:35:17.747+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
